recipes/supervised/meta.yaml

```diff
@@ -0,0 +1,64 @@
+
```

Comment:
```suggestion
```

Reply:
@xhochy  thanks for looking into this. I removed the line